common/parallel/TreeCrawler.go

```diff
@@ -185,6 +185,7 @@ func (c *crawler) readTqueue() {
 		//
 		if len(c.unstartedDirs) > maxQueueDirectories {
 			for len(c.unstartedDirs) > maxQueueDirectories*0.8 {
+				fmt.Printf("Number of directories in unstartedDirs queue reached max limit:%v.", maxQueueDirectories)
```

Comment:
fmt.Printf("Number of directories in unstartedDirs queue reached high water mark (%v): %v", maxQueueDirectories*0.8, len(c.unstartedDirs))